blueprints/validator-test/index.js

```diff
@@ -0,0 +1,38 @@
+var path = require('path');
+var existsSync = require('exists-sync');
+var walkSync = require('walk-sync');
+
+module.exports = {
+  description: 'Generates a validator unit test',
+
+  _getFilesPath: function() {
```

Comment:
Let's name this filesPath to be in line with future versions once that PR lands.